### PR TITLE
Use an index for the pagination in UTXO query

### PR DESCRIPTION
**Pull request by majecty**

### CODE REVIEW

File: src/routers/asset.ts [hunk 1]
```diff
@@ -133,6 +138,27 @@ export function handle(context: IndexerContext, router: Router) {
...
+                            if (!Number.isInteger(lastEvaluatedKey[index])) {
+                                throw new Error("Invalid lastEvalulatedKey");
+                            }
+                        }
+                    }
+                } catch (e) {
+                    next(e);
+                }
+            }
+
```
Comment (majecty) [inline src/routers/asset.ts]: I want to use `Joi` to verify the `lastEvaluatedKey`. However, `Joi` does not support `JSON` string. @joojis Do you have any idea?

Comment (majecty) [reply src/routers/asset.ts]: I tried using express middleware. What do you think about using express middleware for this purpose?

https://github.com/CodeChain-io/codechain-indexer/pull/340/commits/9f15eb261616179106c61ecd26270ab6b01e3a29

Comment (joojis) [reply src/routers/asset.ts]: Doesn't it support writing custom functions like `() -> bool`?

Comment (joojis) [reply src/routers/asset.ts]: Oh, it seems it doesn't.

Comment (joojis) [reply src/routers/asset.ts]: Oops, sorry. There is an `any.custom()`. https://hapi.dev/family/joi/?v=16.1.7#anycustommethod-description

Comment (majecty) [reply src/routers/asset.ts]: Aha. I'll try it. Thanks!

Comment (majecty) [reply src/routers/asset.ts]: `any.custom()` needs version 16 of `joi`. I tried to update the `joi` library but failed.

**Pull request merged**